cpp/dolfinx/function/FunctionSpace.cpp

```diff
@@ -123,7 +123,7 @@ bool FunctionSpace::has_element(const fem::FiniteElement& element) const
 std::vector<int> FunctionSpace::component() const { return _component; }
 //-----------------------------------------------------------------------------
 Eigen::Array<double, Eigen::Dynamic, 3, Eigen::RowMajor>
-FunctionSpace::tabulate_dof_coordinates() const
+FunctionSpace::_internal_tabulate_dof_coordinates(int repeats) const
```

Comment:
What's the significance of the underscore? Could the function be put in an anonymous namespace?

Reply:
Accidental Python naming. I've moved it to anonymous